src/python_bindings/py_data_stats.cpp

```diff
@@ -0,0 +1,167 @@
+    return GetValue(stat);
+}
+
+py::object PyDataStats::GetStandardizedCentralMomentOfDist(size_t index, int number) const {
+    algos::Statistic stat = GetAlgorithm<algos::DataStats>(algorithm_)
+                                    .GetStandardizedCentralMomentOfDist(index, number);
+    return GetValue(stat);
+}
+
+py::object PyDataStats::CalculateCentralMoment(size_t index, int number,
+                                               bool bessel_correction) const {
+    algos::Statistic stat = GetAlgorithm<algos::DataStats>(algorithm_)
+                                    .CalculateCentralMoment(index, number, bessel_correction);
+    return GetValue(stat);
+}
+
+py::object PyDataStats::GetMin(size_t index) const {
+    algos::Statistic stat = GetAlgorithm<algos::DataStats>(algorithm_).GetMin(index);
+    return GetValue(stat);
+}
+
+py::object PyDataStats::GetMax(size_t index) const {
+    algos::Statistic stat = GetAlgorithm<algos::DataStats>(algorithm_).GetMax(index);
+    return GetValue(stat);
+}
+
+py::object PyDataStats::GetSum(size_t index) const {
+    algos::Statistic stat = GetAlgorithm<algos::DataStats>(algorithm_).GetSum(index);
+    return GetValue(stat);
+}
+
+py::object PyDataStats::GetQuantile(double part, size_t index, bool calc_all) {
+    algos::Statistic stat =
+            GetAlgorithm<algos::DataStats>(algorithm_).GetQuantile(part, index, calc_all);
+    return GetValue(stat);
+}
+
+py::object PyDataStats::GetNumberOfZeros(size_t index) const {
+    algos::Statistic stat = GetAlgorithm<algos::DataStats>(algorithm_).GetNumberOfZeros(index);
+    return GetValue(stat);
+}
+
+py::object PyDataStats::GetNumberOfNegatives(size_t index) const {
+    algos::Statistic stat = GetAlgorithm<algos::DataStats>(algorithm_).GetNumberOfNegatives(index);
+    return GetValue(stat);
+}
+
+py::object PyDataStats::GetSumOfSquares(size_t index) const {
+    algos::Statistic stat = GetAlgorithm<algos::DataStats>(algorithm_).GetSumOfSquares(index);
+    return GetValue(stat);
+}
+
+py::object PyDataStats::GetGeometricMean(size_t index) const {
+    algos::Statistic stat = GetAlgorithm<algos::DataStats>(algorithm_).GetGeometricMean(index);
+    return GetValue(stat);
+}
+
+py::object PyDataStats::GetMeanAD(size_t index) const {
+    algos::Statistic stat = GetAlgorithm<algos::DataStats>(algorithm_).GetMeanAD(index);
+    return GetValue(stat);
+}
+
+py::object PyDataStats::GetMedian(size_t index) const {
+    algos::Statistic stat = GetAlgorithm<algos::DataStats>(algorithm_).GetMedian(index);
+    return GetValue(stat);
+}
+
+py::object PyDataStats::GetMedianAD(size_t index) const {
+    algos::Statistic stat = GetAlgorithm<algos::DataStats>(algorithm_).GetMedianAD(index);
+    return GetValue(stat);
+}
+
+size_t PyDataStats::GetNumNulls(size_t index) const {
+    algos::Statistic stat = GetAlgorithm<algos::DataStats>(algorithm_).GetNumNulls(index);
+    return static_cast<size_t>(mo::Type::GetValue<mo::Int>(stat.GetData()));
```

Comment:
Hm, I'm wondering why `GetNumNulls` returns a `Statistic` instead of a plain integer